searchlib/src/vespa/searchlib/features/attributefeature.cpp

```diff
@@ -413,10 +415,25 @@ createTensorAttributeExecutor(const IAttributeVector *attribute, const vespalib:
 
 }
 
-search::fef::FeatureExecutor &
-AttributeBlueprint::createExecutor(const search::fef::IQueryEnvironment &env, vespalib::Stash &stash) const
+void
+AttributeBlueprint::prepareSharedState(const fef::IQueryEnvironment & env, fef::IObjectStore & store) const
+{
+    if (store.get(_attrKey) == nullptr) {
```

Comment:
if the intent is to NOT share attribute lookups between features, consider removing this 'if' check.

Reply:
Intent to share. Fixed.